scripts/ci/assay_runner_lane_check.py

```
@@ -180,6 +220,288 @@ def next_link(header: str) -> str | None:
+        return None, ("gh CLI not available for attestation verification",)
+    with tempfile.TemporaryDirectory(prefix="assay-lane-attestation-") as tmp:
+        root = Path(tmp)
+        manifest_path = root / "manifest.json"
+        bundle_path = root / "attestation-bundle.json"
+        manifest_path.write_bytes(pack.manifest_bytes)
+        bundle_path.write_bytes(pack.bundle_bytes)
+        env = dict(os.environ)
+        if not env.get("GH_TOKEN"):
+            env["GH_TOKEN"] = api.token
+        try:
+            completed = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)